variables.tf

```diff
@@ -1,3 +1,16 @@
+variable "context" {
```

Comment:
This is a breraking change and there are a lot of people depending of this project maybe for a newer version this could be something that could be thought of. If you could elaborate more on the use cases that would be great.